static/js/click_handlers.js

```diff
@@ -192,6 +196,19 @@ exports.initialize = function () {
         $(".tooltip").remove();
     });
 
+    $('body').on('click', '.message_edit_notice', function (e) {
+        var msgid = rows.id($(e.currentTarget).closest(".message_row"));
+        var row = current_msg_list.get_row(msgid);
+        var message = current_msg_list.get(rows.id(row));
+        var message_history_cancel_btn = $('#message-history-cancel');
+
+        popovers.hide_all();
+        message_edit.show_history(message);
+        message_history_cancel_btn.focus();
```

Comment:
What happens if one has edit history disabled in the organization permissions menu?  Might need a conditional for that.

Reply:
Added the `page_params.realm_allow_edit_history` check and also added handlers to remove the on-hover behavior when edit history is disabled.